apps/react/src/features/studios/components/StudioSearch/StudioSearch.tsx

```diff
@@ -0,0 +1,49 @@
+import { IconButton, InputBase, Paper } from '@mui/material';
+import { ChangeEvent, FC, useState } from 'react';
+import SearchIcon from '@mui/icons-material/Search';
+import { useAppDispatch } from '@js-camp/react/store';
+import { resetCursor, setPaginationEvent, setSearchValue } from '@js-camp/react/store/studio/slice';
+import { useSearchParams } from 'react-router-dom';
+
+import styles from './StudioSearch.module.css';
+
+const StudioSearchComponent: FC = () => {
+	const [inputValue, setInputValue] = useState('');
+	const dispatch = useAppDispatch();
+	const [searchParams] = useSearchParams();
+	const searchValue = searchParams.get('search');
+
+	const handleInputValueChange = (event: ChangeEvent) => {
+		setInputValue((event.target as HTMLInputElement).value);
+	};
```

Comment:
You can avoid type assertion if you provide type definitions in a different way:
```suggestion
	const handleInputValueChange: ChangeEventHandler<HTMLInputElement> = event => {
		setInputValue(event.target.value);
	};
```